.github/workflows/publish-release.yml

```
@@ -95,6 +109,7 @@ jobs:
 
       - name: Extract Stable Release Metadata
         id: metadata
+        if: ${{ steps.webhook_secrets_available.outputs.available == 'true' }}
```

actionlint (1.7.12)

[error] property "webhook_secrets_available" is not defined in object type {webhook_secrets_check: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)